### ISSUE TITLE
x509 can't read from stdin

### ISSUE THREAD

**Issue by dcooper16** [openssl/openssl#11871]
The man page for x509 says:
>        -in filename
>            This specifies the input filename to read a certificate from or
>            standard input if this option is not specified.

The following command works with OpenSSL 3.0.0-alpha2:
```
cat cert.pem | openssl x509 -text
```
Using the current master branch on GitHub, the same command returns an error:
```
Could not open file or uri  for loading Certificate
80:1B:4D:82:29:7F:00:00:error:system library:file_open:No such file or directory:crypto/store/loader_file.c:882:calling stat()
Unable to load Certificate
```
The current master branch on GitHub does work, however, if the command line is changed to:
```
cat cert.pem | openssl x509 -text -in /dev/stdin
```
My guess is that PR #11755 is the reason that the command no longer reads from standard input if the `-in` option is not specified.

**Comment by levitte** [comment]
Yup, there's an unfortunate 0 where there should have been a 1.  Fix coming up!

**Comment by levitte** [comment]
Fix in #11873
